### ISSUE TITLE
Redis Windows 6.2.5 下载

### ISSUE THREAD

**Issue by osthink** [tporadowski/redis#114]
Redis Windows 最新版
永久更新地址：https://www.renren.io/detail/14294

最近编译时间：
+-----------------------------------------+
Redis 6.2.5 Windows（x64） 2021-08-04 23:28

**Comment by tporadowski** [comment]
Looks interesting - it's using Cygwin from what I can see in the screenshots. It would be nice to benchmark it against native Windows version, but 6.x is still "under construction"...

**Comment by heylogt** [comment]
Redis compiled using cygwin and other simulation environments cannot be used in production environments, but only in test environments. It has been actually tested, and there will be problems with the number of connections. When the number of connections exceeds 200, the server will make errors

**Comment by junchuanwang** [comment]
> Redis compiled using cygwin and other simulation environments cannot be used in production environments, but only in test environments. It has been actually tested, and there will be problems with the number of connections. When the number of connections exceeds 200, the server will make errors

I don't think you can simply generalize the problem as "compiled using cygwin and other simulation environments".

Prove it.

**Comment by yangyiqian** [comment]
Redis compiled in cygwin mode calls the DLL file of windows platform. However, in the case of pressure measurement, there will be performance problems, and the performance attenuation will be very large. It is really not recommended for use in a production environment.

It is recommended to deploy in Linux environment independently.

Or use a version compiled specifically for the windows platform.
@junchuanwang

**Comment by junchuanwang** [comment]
> Redis compiled in cygwin mode calls the DLL file of windows platform. However, in the case of pressure measurement, there will be performance problems, and the performance attenuation will be very large. It is really not recommended for use in a production environment.
> 
> It is recommended to deploy in Linux environment independently.
> 
> Or use a version compiled specifically for the windows platform.
> @junchuanwang

That would make sense, thank you

**Comment by VAllens** [comment]
@osthink Can you upload to Other Cloud?

**Comment by X-Lucifer** [comment]
The available version of windows compiled from reidis source code is based on msys2 and cygwin. It is consistent with the official redis website. It can be run directly or installed as a Windows service.
https://github.com/X-Lucifer/winredis

**Comment by SamuelMarks** [comment]
FYI: As per #104

> Porting of Redis 6 is still in progress on [win-6.0 branch](https://github.com/tporadowski/redis/tree/win-6.0)

(though last update was in Feb)

**Comment by VAllens** [comment]
I have given up on installing and running redis directly on windows server.
I recommend deploying redis to docker.
